static/org.openpsa.widgets/ui.js

```diff
@@ -139,27 +147,26 @@ const org_openpsa_layout = {
 
         providers.forEach(function(provider) {
             li_class = 'provider';
-            if (current === provider.identifier) {
+            if (config.current === provider.identifier) {
                 li_class += ' current';
                 enable_provider(provider);
             }
 
             $('<li class="' + li_class + '">' + provider.placeholder + '</li>')
                 .data('provider', provider)
                 .click(function() {
-                    var old_item = $('#org_openpsa_search_providers .current'),
-                        query = $('#org_openpsa_search_query');
+                    var old_item = form.find('.org_openpsa_search_providers .current');
```

Comment:
if this changes from id to classname, this also has to change: https://github.com/flack/openpsa/search?q=org_openpsa_search_trigger